cfpb/sbl-frontend
# Removed 'Additional Details' from Complete your User Profile & Update your Financial Institution Profile

closes #577 

## Note
- Zod schema and formatting helper functions will be kept in -- incase of future use


## Changes
- chore(additional_details): removed just the UI elements

## How to test this PR

1. Test Complete your User Profile (No Associations) -- verify requests still go through with the proper data body
2. Test Update your Financial Institution Profile -- verify requests still go through with the proper data body

## Screenshots
<img width="407" alt="Screenshot 2024-05-29 at 3 58 24 AM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/3f0ebf30-8cb0-411c-864d-02afc65ed33d">
<img width="604" alt="Screenshot 2024-05-29 at 3 55 05 AM" src="https://github.com/cfpb/sbl-frontend/assets/13324863/e48573f0-f377-4397-aa06-7e7e6d1803e5">
